docs/source/technical_overview.rst

```diff
@@ -350,27 +350,40 @@ This approach has several advantages over an inheritance-based model:
 The Reduction Process
 ---------------------
 
-The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`
-which take arbitrary |Models| and additional data (e.g. the reduced
-basis) to create reduced |Models|. If proper offline/online
-decomposition is achieved by the reductor, the reduced |Model| will
-not store any high-dimensional data. Note that there is no inherent distinction
-between low- and high-dimensional |Models| in pyMOR. The only
+The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`,
+where each reductor takes arbitrary |Models| of associated type, and additional
+data (e.g. the reduced basis) to create reduced |Models|. For instance, the
+:class:`~pymor.reductors.basic.StationaryRBReductor` and
+:class:`~pymor.reductors.basic.InstationaryRBReductor` reductors can reduce any
+:class:`~pymor.models.basic.StationaryModel` and :class:`~pymor.models.basic.InstationaryModel`,
+while the :class:`~pymor.reductors.bt.BTReductor` reduces
+:class:`~pymor.models.iosys.LTIModel` (just to name a few).
+
+If proper offline/online decomposition is achieved by the reductor, the reduced
+|Model| will not store any high-dimensional data. Note that there is no inherent
+distinction between low- and high-dimensional |Models| in pyMOR. The only
 difference lies in the different types of operators, the |Model|
 contains.
 
 This observation is particularly apparent in the case of the classical
 reduced basis method: the operators and functionals of a given discrete problem
 are projected onto the reduced basis space whereas the structure of the problem
 (i.e. the type of |Model| containing the operators) stays the same.
-pyMOR reflects this fact by offering with :class:`~pymor.reductors.basic.GenericRBReductor`
-a generic algorithm which can be used to RB-project any model available to pyMOR.
-It should be noted however that this reductor is only able to efficiently
-offline/online-decompose affinely |Parameter|-dependent linear problems.
-Non-linear problems or such with no affine |Parameter| dependence require
+pyMOR reflects this fact by offering with :meth:`~pymor.algorithms.projection.project`
```

Comment:
I don't think this makes sense logically. The existence of `project` does not show that models do not change their structure when being projected. Maybe just remove the paragraph about the structure and go on with saying that most of pyMOR's reductors internally use `project` as a generic algorithm for projecting arbitrary operators.